test/verify/run_verify.hpp

```diff
@@ -24,9 +24,11 @@
 #ifndef MIGRAPHX_GUARD_TEST_RUN_VERIFY_HPP
 #define MIGRAPHX_GUARD_TEST_RUN_VERIFY_HPP
 
+#include "verify_program.hpp"
```

Comment:
We can forward declare the `program_info` class instead of including the header.

Reply:
Done.